src/s3/commands/uploadFile.ts

```diff
@@ -26,51 +38,103 @@ export interface FileSizeBytes {
 }
 
 /**
- * Uploads a file to the bucket or folder represented by the given node.
- *
- * Prompts the user for the file location.
- * Shows the output channel with "upload started" message.
- * Uploads the file (showing a progress bar).
- * Shows the output channel with "upload completed" message.
- * Refreshes the node.
- *
- * Node that the node is reset to displaying its first page of results.
- * The file that is uploaded won't necessary fall on the first page.
- * The user may need to load more pages to see the uploaded file reflected in the tree.
+ * Wizard to upload a file.
+ * If the S3BucketNode | S3FolderNode is given, it asks for the file then uploads to node
+ * If the node is not give, it asks for the file and also for a bucket, then uploads
+ * 
+ * @param s3Client - account to upload the file to
```

Comment:
no need for hyphen here. (inconsistent with existing codebase, and it's already highlighted/rendered with a separator)